src/UI/Internal/Tables/FiltersView.elm

```diff
@@ -194,14 +194,18 @@ headerAttrs isApplied =
         ++ workingTheme
 
 
+
+-- Standard size used for headers
+
+
 textSize : Int
 textSize =
     12
 
 
-size : Size
-size =
-    Size.Small
+contextSize : Size
+contextSize =
+    Size.ExtraSmall
```

Comment:
why "context"?

Reply:
Cause this is the "contextual size" of the header. Applying to all components on it.